recipes/serpent/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.12" %}
+
+package:
+  name: serpent
```

Comment:
The documentation suggests that the package name is lower case.

--Bill

On Sun, May 1, 2016 at 2:19 PM, Filipe <notifications@github.com> wrote:

> In recipes/serpent/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/490#discussion_r61690613>
> :
>
> > @@ -0,0 +1,37 @@
> > +{% set version = "1.12" %}
> > +
> > +package:
> > +  name: serpent
>
> The default channel use all lower case. I am slightly inclined to respect
> the original name as-is. But IMO that is no big deal as both conda and PyPI
> are case insensitive.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/490/files/10e1157648cfa841d1ab2e9628426db00b161c35#r61690613>
>